guacamole-common-js/src/main/webapp/modules/Keyboard.js

```diff
@@ -625,6 +653,12 @@ Guacamole.Keyboard = function(element) {
     }
 
     function keysym_from_keycode(keyCode, location) {
+
+        // Map Clear on Mac to NumLock
+        if (keyCode === 12 && is_mac) {
```

Comment:
Mac doesn't have Num Lock on the keyboard, which makes it hard to turn
on/off. Usually Clear (which is at the same location as Num Lock in a PC
keyboard) is mapped to Num Lock for convenience.
On Sep 20, 2015 2:23 PM, "Mike Jumper" <notifications@github.com> wrote:

> In guacamole-common-js/src/main/webapp/modules/Keyboard.js
> <https://github.com/glyptodon/guacamole-client/pull/253#discussion_r39934850>
> :
>
> > @@ -625,6 +653,12 @@ Guacamole.Keyboard = function(element) {
> >      }
> >
> >      function keysym_from_keycode(keyCode, location) {
> > +
> > +        // Map Clear on Mac to NumLock
> > +        if (keyCode === 12 && is_mac) {
>
> Why?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/glyptodon/guacamole-client/pull/253/files#r39934850>.
>